profile/README.md

```diff
@@ -1 +1,45 @@
-<img src="https://user-images.githubusercontent.com/73097560/133384648-1d3986fa-e388-4f36-a094-f583182ccef6.png" width="100%">
+![Banner](https://user-images.githubusercontent.com/72812470/134735139-8dd74c7f-82b3-4a80-8e28-205ba04ce345.png)
+<br>
+<p align="center">
+<a href= "https://github.com/web3community/web3community.github.io/issues" alt = "Open-issues">
+  <img src = "https://img.shields.io/github/issues/web3community/web3community.github.io" target="blank" />
+</a>
+<a href="https://discord.gg/BHa8Xn9Cgf" alt="community-discord-server">
+   <img src="https://img.shields.io/discord/835424705410236427?logo=discord&?style=for-the-badge" target="blank" />
+</a>
+<a href= "https://github.com/web3community" alt = "Stars">
+  <img src = "https://img.shields.io/github/stars/web3community/blockchain-dev-path?style=social" target="blank" />
+</a>
+<a href="https://twitter.com/web3community" alt="Follow Web3Community on Twitter">
+<img src="https://img.shields.io/twitter/follow/web3community?label=web3community&style=social" /></a>
+<a href="https://github.com/web3community/start-here" alt="License">
+<img src="https://img.shields.io/github/license/web3community/start-here.svg" /></a>
+<a href="https://github.com/web3community" alt="Meshery contributors">
+<img alt="GitHub contributors" src="https://img.shields.io/github/contributors-anon/web3community/web3community.github.io"></a>
+<a href= "https://github.com/web3community" alt = "Closed-issues">
+  <img src = "https://img.shields.io/github/issues-closed-raw/web3community/web3community.github.io" target="blank" />
```

Comment:
I think we don't need to put issues, license, contributors, closed issues badges here?

I suggest removing it.